zap/src/main/java/org/apache/commons/httpclient/URI.java

```diff
@@ -59,14 +59,14 @@
  */
 /**
  * The interface for the URI(Uniform Resource Identifiers) version of RFC 2396.
- * This class has the purpose of supportting of parsing a URI reference to
+ * This class has the purpose of supporting of parsing a URI reference to
  * extend any specific protocols, the character encoding of the protocol to 
  * be transported and the charset of the document.
  * <p>
  * A URI is always in an "escaped" form, since escaping or unescaping a
  * completed URI might change its semantics.  
  * <p>
- * Implementers should be careful not to escape or unescape the same string
+ * Implementors should be careful not to escape or unescape the same string
```

Comment:
In principle, this is the wrong spelling, but the wrong spelling is the vast majority, so I've decided to side with it.

Sometimes I do, sometimes I don't.